lib/datadog/tracing/configuration/ext.rb

```diff
@@ -37,7 +37,7 @@ module Distributed
           # Sets both extract and inject propagation style tho the provided value.
           # Has lower precedence than `DD_TRACE_PROPAGATION_STYLE_INJECT` or
           # `DD_TRACE_PROPAGATION_STYLE_EXTRACT`.
-          ENV_PROPAGATION_STYLE_EXTRACT = 'DD_TRACE_PROPAGATION_STYLE'.freeze
+          ENV_PROPAGATION_STYLE = 'DD_TRACE_PROPAGATION_STYLE'.freeze
```

Comment:
I introduced this constant recently (a change still not released), but copy/pasted the constant name. This fixes that.